templates/beego/files/controllers/default.go

```diff
@@ -0,0 +1,15 @@
+package controllers
+
+import (
+	"github.com/astaxie/beego"
+)
+
+type MainController struct {
+	beego.Controller
+}
+
+func (c *MainController) Get() {
+	c.Data["Website"] = "beego.me"
+	c.Data["Email"] = "astaxie@gmail.com"
```

Comment:
Who is astaxie, and why are we trying to spam him?

Reply:
I can remove that, this is a part of the standard Welcome to Beego page. I can remove everything but the website.

Reply:
I don't know why he wants so much spam, but yeah, let's remove the email line at least to be good netizens. :smile: